src/main/java/org/reactome/addlinks/EnsemblBiomartUtil.java

```diff
@@ -0,0 +1,148 @@
+package org.reactome.addlinks;
+
+import org.json.simple.JSONArray;
+import org.json.simple.JSONObject;
+import org.json.simple.parser.JSONParser;
+import org.json.simple.parser.ParseException;
+
+import java.io.FileInputStream;
+import java.io.FileReader;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.Properties;
+import java.util.stream.Collectors;
+
+public class EnsemblBiomartUtil {
+
+
+    public static final String proteinToGenesSuffix = "_proteinToGenes";
+    public static final String proteinToTranscriptsSuffix = "_proteinToTranscripts";
+    public static final String transcriptToMicroarraysSuffix = "_transcriptToMicroarrays";
+    public static final String uniprotToProteinsSuffix = "_uniprotToProteins";
```

Comment:
For a class level constant names, UPPER_CASE_WITH_UNDERSCORES is preferred :)

Reply:
Done